### PR TITLE
[CI:DOCS] Update release notes for v1.9.0-RC1

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat @baude @rhatdan PTAL

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
 
+### Misc
+- All uses of the `--timeout` flag have been switched to prefer `--time`. The `--timeout` flag will continue to work, but man pages and `--help` will use the `--time` flag instead
+- The default infra image for pods has been upgraded to `k8s.gcr.io/pause:3.2` (from 3.1) to address a bug in the architecture metadata for non-AMD64 images
+- The `slirp4netns` networking utility in rootless Podman now uses Seccomp filtering where available for improved security
+- Updated Buildah to v1.14.7
+- Updated containers/storage to v1.18.2
+- Updated containers/image to v5.4.3
+- Updated containers/common to v0.8.1
+
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:26]: we bumped opencontainers/selinux to v1.5.1, should we list it and maybe containers/common v0.8.1 too?

**Comment by TomSweeneyRedHat** [comment]
One questions for consideration, otherwise LGTM.

**Comment by rhatdan** [comment]
We need to wait for the make vendor/tidy changes by @mtrmac

**Comment by mheon** [comment]
I think that will result in a fresh Buildah minor version, so should we hold this for that?

**Comment by rhatdan** [comment]
Yes.  Hold

**Comment by mheon** [comment]
Pushed some major additions. @TomSweeneyRedHat @edsantiago PTAL

**Comment by edsantiago** [comment]
LGTM but I need to confess that I haven't cross-checked against `git log --graph v1.8.2..master` - I got scared off by line count.

**Comment by TomSweeneyRedHat** [comment]
We're bumping Buildah to v1.14.8 and I think we're respinning for that.  Might want to bump that in here too.

**Comment by mheon** [comment]
@TomSweeneyRedHat Fixed

Anyone want to re-lgtm?

**Comment by edsantiago** [comment]
Verified [the diff](https://github.com/containers/libpod/compare/36399517887adefe246e5ef4401e444cdd3dbda5..00deffd).

/lgtm
/hold

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**